lisp/ess-custom.el

```diff
@@ -887,7 +887,7 @@ See `ess-style-alist' for for an overview of ESS indentation."
      (ess-indent-function-declaration . t)
      (ess-indent-from-lhs . t)
      (ess-indent-with-fancy-comments . t))
-    
```

Comment:
It's all about synchronization with others. If we can activate that settings for all files in ESS directory that would be good. 

 I was not using `whitespace-cleanup` myself because of the inability to set it per project basis. But now I see that there is a third party [`whitespace-cleanup-mode`](https://github.com/purcell/whitespace-cleanup-mode) and also [`ws-butler`](https://github.com/lewang/ws-butler).  

I will add a setting to `.dir-locals`.